repo.go

```diff
@@ -76,11 +76,11 @@ func (r *Repo) Init(consistentSnapshot bool) error {
 	}
 	root := data.NewRoot()
 	root.ConsistentSnapshot = consistentSnapshot
-	err = r.setMeta("root.json", root)
-	if err != nil {
-		return err
-	}
-	_, err = r.GenKey("root")
+	return r.setMeta("root.json", root)
```

Comment:
Can an setMeta error be recovered from?

Also, what do we think of two argument methods having a method call as return? I've been thinking about that a bit, and haven't decided how I feel about it.